src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  margin-left: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg);
+    background-repeat: no-repeat;
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background: url(../images/star-active.svg);
+      background-repeat: no-repeat;
```

Comment:
Do you really need to set `background-repeat` here?

Reply:
I guess so, if i set it only in stars__star block, my active star images would repeat 

![image](https://github.com/mate-academy/layout_catalog/assets/74732843/06c5c129-fced-4285-ab52-8494b35e96aa)